CalibMuon/DTCalibration/python/dtCalibValidationFromMuons_cfg.py

```diff
@@ -0,0 +1,42 @@
+import FWCore.ParameterSet.Config as cms
+
+process = cms.Process("Validation")
+
+process.load("FWCore.MessageService.MessageLogger_cfi")
+process.MessageLogger.cerr.threshold = 'INFO'
+
+process.load("CalibMuon.DTCalibration.dt_offlineAnalysis_common_cff")
+process.load("DQMServices.Core.DQM_cfg")
+
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring()
```

Comment:
@saghosh is this supposed to be just a template?

Reply:
@fabiocos yes, this is a template that is imported in the workflow and then certain variables are updated as required for a specific action in the workflow